### PR TITLE
Feat/video label

### PR DESCRIPTION
**Pull request by nikhedonia**
Implements video label as a seperate package.
In order to avoid magic numbers when using the video-icon inside the video label, I made width of icons optional and compute the width from the aspect ratio and height.

videolabel

reference: https://www.thetimes.co.uk/past-six-days/2018-03-06/sport/weekend-review-mourinhos-tumble-richarlisons-tantrum-and-dioufs-spat-tfsfwtbkf

### CODE REVIEW

File: packages/video-label/__tests__/shared.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "react-native";
+import React from "react";
+import renderer from "react-test-renderer";
+import VideoLabel from "../video-label";
+
+module.exports = () => {
```
Comment (colincclark) [inline packages/video-label/__tests__/shared.js]: `export default`

File: packages/video-label/style/index.android.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import sharedStyles from "./shared";
+
+const styles = {
+  title: {
+    ...sharedStyles.title
```
Comment (colincclark) [inline packages/video-label/style/index.android.js]: these index files do not seem to add anything...

File: packages/video-label/video-label.android.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    .split("")
+    .join("\u200A");
+
+const VideoLabel = ({ title, color }) => (
+  <View style={{ flexDirection: "row", alignItems: "center" }}>
+    <IconVideo height={styles.title.fontSize} fillColor={color} />
+    <Text style={[styles.title, { color, marginLeft: 5 }]}>
+      {beautifyTitle("VIDEO")}
+    </Text>
+    {title ? (
```
Comment (colincclark) [inline packages/video-label/video-label.android.js]: `&&` instead of a ternary?

File: packages/video-label/video-label.android.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      <Text style={[styles.title, { color, marginLeft: 3 }]}>|</Text>
+    ) : null}
+    {title ? (
```
Comment (colincclark) [inline packages/video-label/video-label.android.js]: `&&`

File: packages/video-label/video-label.stories.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "Section",
+        invert(colours.section),
+        colours.section.default
+      )}
+    />
+  ))
+  .add("VideoLabel with title", () => (
+    <VideoLabel
+      title="SWIMMING"
+      color={select(
```
Comment (colincclark) [inline packages/video-label/video-label.stories.js]: nice knobs

File: packages/video-label/video-label.android.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { View, Text } from "react-native";
+import { IconVideo } from "@times-components/icons";
+import PropTypes from "prop-types";
+import styles from "./style";
+
+// apply transformations to add uppercase and letter spacing.
+// letterSpacing CSS prop does not work on android:
+// https://github.com/facebook/react-native/pull/13199
+const beautifyTitle = title =>
```
Comment (aronshamash) [inline packages/video-label/video-label.android.js]: maybe we can now get rid of this in light of https://github.com/facebook/react-native/pull/17398. Just need to check

Comment (nikhedonia) [reply packages/video-label/video-label.android.js]: Not part of react-native-0.53.3

File: packages/video-label/video-label.android.js (hunk 3 above)
Comment (nikhedonia) [reply packages/video-label/video-label.android.js]: to my surprise this does not seem to work:

```
Unexpected text node: . A text node cannot be a child of a <View>.
```

File: packages/video-label/video-label.android.js (hunk 6 above)
Comment (aronshamash) [reply packages/video-label/video-label.android.js]: ok

File: packages/video-label/video-label.js [hunk 7]
```diff
@@ -8,9 +8,9 @@ const VideoLabel = ({ title, color }) => (
   <View style={{ flexDirection: "row", alignItems: "center" }}>
     <IconVideo height={styles.title.fontSize} fillColor={color} />
     <Text style={[styles.title, { color, marginLeft: 5 }]}>VIDEO</Text>
-    {title ? <Text style={[styles.title, { marginLeft: 3 }]}>|</Text> : null}
+    {title ? <Text style={[styles.title, { marginLeft: 5 }]}>|</Text> : null}
     {title ? (
-      <Text style={[styles.title, { color, paddingLeft: 3 }]}>{title}</Text>
+      <Text style={[styles.title, { color, paddingLeft: 5 }]}>{title.toUpperCase()}</Text>
```
Comment (aronshamash) [inline packages/video-label/video-label.js]: better

File: packages/icons/icons/video.js [hunk 8]
```diff
@@ -6,8 +6,9 @@ import iconPropTypes from "./proptypes";
 const viewBox =
   "0.15463916957378387 0.049614034593105316 23.59917640686035 13.728596687316895";
 
+const ratio = 1.72;
```
Comment (aronshamash) [inline packages/icons/icons/video.js]: In `twitter.js` we using a ratio here we using a decimal. Can use `43 / 25` for video and `75 / 60` for twitter. Other than this it seem good to merge

**Pull request merged**